modules/standard/fun/ding_controller.py

```diff
@@ -0,0 +1,108 @@
+import random
+
+from core.command_param_types import SignedInt
+from core.decorators import command, instance
+from core.registry import Registry
+
+
+@instance()
+class DingController:
+    """
+        Port of Budabot fun module
+    """
+
+    def inject(self, registry: Registry):
+        self.command_alias_service = registry.get_instance(
+            "command_alias_service")
+
+    def start(self):
+        self.command_alias_service.add_alias("levelup", "ding")
```

Comment:
is this command called !levelup in another popular bot? if so, I am OK having it as a default alias, but otherwise we shouldn't define a default alias

Reply:
To be honest I dont think it is.

I'll remove it. 👍